regulation/node.py

```diff
@@ -73,7 +73,7 @@ def to_json(self):
         node_dict['text'] = self.text
         if self.title and self.title != '':
             node_dict['title'] = self.title
-        if self.marker:
+        if self.marker is not None:
```

Comment:
On else, `marker` doesn't exist in the dictionary. 

`marker` [defaults to `None`](https://github.com/willbarton/regulations-xml-parser/blob/ensure-markerless-json/regulation/node.py#L38).